pkg/kubelet/nodeshutdown/nodeshutdown_manager_others.go

```diff
@@ -1,5 +1,5 @@
-//go:build !linux
-// +build !linux
+//go:build !linux && !windows
+// +build !linux,!windows
```

Comment:
nit: update the comment below

> // NewManager returns a fake node shutdown manager for non linux platforms.

Maybe just say "for unsupported platforms"?

Reply:
updated